Makefile

```diff
@@ -120,10 +122,11 @@ e2e-substitutions: $(ENVSUBST) $(subst $(E2E_TEMPLATES_DIR),$(E2E_ENVSUBST_DIR),
 	$(ENVSUBST) < $(E2E_CONF_FILE) > $(E2E_CONF_FILE_ENVSUBST)
 
 .PHONY: e2e-tests
+e2e-tests: CONTAINER_RUNTIME?=docker ## Env variable can override this default
 e2e-tests: e2e-substitutions ## This target should be called from scripts/ci-e2e.sh
-	docker pull quay.io/metal3-io/cluster-api-provider-metal3
-	docker pull quay.io/metal3-io/baremetal-operator
-	docker pull quay.io/metal3-io/ip-address-manager
+	for image in $(E2E_CONTAINERS); do \
+		$(CONTAINER_RUNTIME) pull $$image; \
```

Comment:
```suggestion
		$(CONTAINER_RUNTIME) pull $image; \
```

Reply:
This is necessary to espace $ in the Makefile